### DISCUSSION TITLE
I need help displaying the validation error of global validation.

### DISCUSSION THREAD

**Discussion by trupen1010** [livewire/livewire#9013] [1. Help]
We can apply global validation using the following:
https://livewire.laravel.com/docs/uploads#global-validation

However, the document does not explain how we can manage that validation and display it on the site for user experience.

Anyone can Explain How can do that.

**Comment by marchershey** [comment]
- https://github.com/livewire/livewire/discussions/4644
- https://github.com/livewire/livewire/issues/1546
- https://github.com/livewire/livewire/issues/1524
